csrc/abstract_tensor_schedule.h

```diff
@@ -0,0 +1,23 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <abstract_tensor.h>
+#include <ir/internal_base_nodes.h>
+#include <val_graph.h>
+
+namespace nvfuser {
+
+//! Apply the transformations found in an AbstractTensor to a concrete
+//! TensorView.
+void applyAbstractSchedule(
+    const AbstractTensor& abstract,
+    TensorView* concrete,
+    ValGraph* graph = nullptr);
```

Comment:
Maybe `applyAbstractTransforms` would be a better name?

Reply:
+1. That seems more consistent.

Reply:
Done